### PR TITLE
iperf3: add page

### PR DESCRIPTION
**Pull request by chub16**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/iperf3.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# iperf3
+
+> Iperf3 — traffic generator for testing network bandwidth
```
Comment (pxgamer) [inline pages/common/iperf3.md]: Not sure `Iperf3` is needed here. Perhaps just `A traffic generator for testing network bandwidth.`?

Also, could you add a full stop to the end (as noted by the linter).

File: pages/common/iperf3.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Run iperf3 as a server
```
Comment (pxgamer) [inline pages/common/iperf3.md]: Could you add a colon to the end here? As with the other example descriptions.

File: pages/common/iperf3.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Start bandwidth test
+
+`iperf3 -c {{server}}`
+
+- Start bandwidth test in 5 threads
+
+`iperf3 -c {{server}} -P5`
+
+- Start bandwidth test in reverse mode
```
Comment (pxgamer) [inline pages/common/iperf3.md]: What is reverse mode? 🤔

File: pages/common/iperf3.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# iperf3
+
+> Iperf3 — traffic generator for testing network bandwidth
+
+- Run iperf3 as a server
+
+`iperf3 -s`
+
+- Run iperf3 server on port 5209
```
Comment (pxgamer) [inline pages/common/iperf3.md]: Perhaps tokenize the port. Such as `Run an iperf3 server on a specific port:` and then
`iperf3 -s -p{{port}}`

File: pages/common/iperf3.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`iperf3 -s -p5209`
+
+- Start bandwidth test
+
+`iperf3 -c {{server}}`
+
+- Start bandwidth test in 5 threads
+
+`iperf3 -c {{server}} -P5`
```
Comment (pxgamer) [inline pages/common/iperf3.md]: Perhaps tokenize the threads, as `-P{{threads}}`

File: pages/common/iperf3.md (hunk 3 above)
Comment (chub16) [reply pages/common/iperf3.md]: Reverse mode means a different direction of the test. The server sends data to the client.
Perhaps in this case it's better to write `Reverse direction of the test. Server sends data to the client:`?

**Comment by pxgamer** [comment]
Hi @chub16, sorry could you add a full stop (period) to the end of the first description?

**Comment by pxgamer** [comment]
Looks alright to me.

File: pages/common/iperf3.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- Run an iperf3 server on a specific port:
+
+`iperf3 -s -p{{port}}`
+
+- Start bandwidth test:
+
+`iperf3 -c{{server}}`
+
+- Run iperf3 in multiple threads:
```
Comment (agnivade) [inline pages/common/iperf3.md]: Isn't "threads" the incorrect term for this ? `-P` specifies the no. of connections to make to the server. A single thread can also create multiple connections.

Comment (pxgamer) [reply pages/common/iperf3.md]: Perhaps `multiple parallel streams`?

Comment (sbrl) [reply pages/common/iperf3.md]: Sounds good to me, @pxgamer. Ping @agnivade

Comment (agnivade) [reply pages/common/iperf3.md]: Sure, sounds good.

**Pull request merged**